src/components/CopyURL.js

```diff
@@ -0,0 +1,82 @@
+import styled from "styled-components";
+import { transitionCss } from "../styles/styleUtils";
+import Input from "./Input";
+
+const ButtonStyle = styled.button`
+  background: ${(p) => (p.theme.colors.secondary)};
+  color: ${(p) => (p.theme.colors.text)};
+  opacity: ${p => p.disabled ? "0.5" : "1"};
+  cursor: ${p => p.disabled ? "not-allowed" : "auto"};
+  font-size: ${(p) => p.size ? p.theme.fontSizes[p.size] : p.theme.fontSizes.default};
+  font-family: ${(p) => p.theme.fonts};
+  font-weight: ${(p) => p.theme.fontWeight};
+  border: ${(p) => p.type === "text" ? "none" : "default"};
+  border-radius: 5px;
+  padding: 0.25em 0.25em;
+  ${transitionCss}
+  :hover {
+    opacity: 0.65;
+  }
+`;
+
+const inputSize = {
+    medium: 24,
+    large: 36,
+    default: 24,
+  };
+
+const InputBox = styled.input`
+  width: ${(p) => (inputSize[`${p.size}`] ?? inputSize["default"]) * 9}px;
+  height: ${(p) => inputSize[`${p.size}`] ?? inputSize["default"]}px;
+  font-size: ${(p) => inputSize[`${p.size}`] ?? inputSize["default"]}px;
+  font-family: ${(p) => p.theme.fonts};
+  font-weight: ${(p) => p.theme.fontWeight};
+  color: ${(p) => p.fontColor ?? "black"};
+  cursor: ${(p) => (p.disabled ? "not-allowed" : "text")};
+  padding: 10px;
+  border: ${(p) => p.border ?? "1px solid black"};
+  border-radius: 3px;
+  
+  :placeholder{
+    color: ${(p) => p.placeholderColor ?? "black"}
+  }
+  :disabled {
+    background: #EEEEEE;
+  }
+`;
```

Comment:
Is there a difference between this and the one from `Input.js`? Why can't we just use `<Input>...</Input>`?

Reply:
i'm not sure how it all works out in the end, but since this is a non-editable field for copying the URL (i'm assuming users aren't allowed to edit the URL), should it be an input component o should we make a separate non-editable text field for this?

Reply:
whichever is easier. you could also use something like `<Input placeholder={url} disabled />` if that works. (idk if that is valid but could give it a shot)

Reply:
but yeah try just `import Input from 'components/input`

Reply:
i tried importing from input earlier but the sizing didn't work but lemme try again hehe

Reply:
ok resolved it thanks frands :))